pysqa/wrapper/flux.py

```diff
@@ -19,7 +19,12 @@ def get_queue_status_command(self):
     @staticmethod
     def get_job_id_from_output(queue_submit_output):
         result = subprocess.run(
-            ['flux', 'job', 'id', queue_submit_output.splitlines()[-1].rstrip().lstrip().split()[-1]],
-            stdout=subprocess.PIPE
+            [
+                "flux",
+                "job",
+                "id",
+                queue_submit_output.splitlines()[-1].rstrip().lstrip().split()[-1],
+            ],
+            stdout=subprocess.PIPE,
```

Comment:
Maybe there is an easier way to convert the job ID to an integer ID https://github.com/flux-framework/flux-core/issues/4491

Reply:
Yes, if you can `import flux` then `flux.job.JobID(string)` should convert a Flux jobid in any format to a `JobID` which is a subclass of `int`. (So you could call `int()` on it)

Reply:
@grondo Somehow I get an `AttributeError: module 'flux' has no attribute 'job'` error

Reply:
I was able to fix it by importing `from flux.job import JobID`